quantum/quantum.c

```diff
@@ -103,6 +103,56 @@ bool process_record_quantum(keyrecord_t *record) {
           return false;
       }
       break;
+	#ifdef RGBLIGHT_ENABLE
+	case RGB_TOG:
+		if (record->event.pressed) {
+			rgblight_toggle();
+			return false;
```

Comment:
These `return false;` statements should really go outside the if block - we don't want them to be processed (what returning false does) either way.